### PR TITLE
Update code-splitting.md

### PR DESCRIPTION
**Pull request by EddyVinck**
I updated this part of the documentation to reflect the current demo project.

### CODE REVIEW

**Comment by cane-lagerwaard** [comment]
Preview: https://deploy-preview-68--falcon-docs.netlify.com/docs/falcon-client/code-splitting

File: docs/falcon-client/code-splitting.md [hunk 1]
```diff
@@ -2,34 +2,47 @@
...
-    {
-      /// ...
-    }
+    // Your other routes...
   </Switch>
 )
 ```
 
-For more information see [this](https://github.com/ctrlplusb/react-async-component)
+When you create your project using [`create-falcon-app`](https://falcon.deity.io/docs/getting-started/installation#create-falcon-app-cli-tool) Falcon provides a wrapper function for `loadable`. This wrapper function shows a spinner while the page is loading to indicate to the user that the contents of the page are currently being fetched. Using this wrapper function is optional, but it is recommended to provide a fallback state to `loadable` if you choose to write your own.
```
[the remark this answers is not in the document]
Comment (EddyVinck) [reply docs/falcon-client/code-splitting.md]: I updated the file with your suggestions

[the remark this answers is not in the document]
Comment (Borales) [reply docs/falcon-client/code-splitting.md]: 😢 without committing them

File: docs/falcon-client/code-splitting.md [hunk 2]
```diff
@@ -2,34 +2,47 @@
...
+import loadable from '@loadable/component';
+import { Loader } from '@deity/falcon-ecommerce-uikit';
+
+export default component =>
+  loadable(component, {
+    fallback: <Loader />
+  });
+```
+
+For more information see [`loadable` on GitHub](https://github.com/smooth-code/loadable-components)
```
Comment (Borales) [inline docs/falcon-client/code-splitting.md]:
```suggestion
For more information see [`loadable` on GitHub](https://github.com/smooth-code/loadable-components).
```

File: docs/falcon-client/code-splitting.md [hunk 3]
```diff
@@ -2,34 +2,47 @@
...
 ---
 
-This feature splits your code into various bundles which are loaded on demand. It is used to achieve smaller bundles and control resource load prioritization which have a major impact on load time. To specify the splitting point you need to import modules in an asynchronous manner. To make it working with React, Falcon Client uses [async component](#async-components).
+This feature splits your code into various bundles which are loaded on demand. It is used to achieve smaller bundles and control resource load prioritization. This has a major impact on load time. To specify the splitting point you need to import modules in an asynchronous manner. To make it work with React, Falcon Client uses [loadable](#loadable).
 
-### Async components
+### Loadable
 
-Async components is based on [react-async-component](https://github.com/ctrlplusb/react-async-component)
+Asynchronous components are based on [loadable](https://github.com/smooth-code/loadable-components)
```
[the remark this answers is not in the document]
Comment (Borales) [reply docs/falcon-client/code-splitting.md]: @EddyVinck what about this one?

[the remark this answers is not in the document]
Comment (EddyVinck) [reply docs/falcon-client/code-splitting.md]: I did that here https://github.com/deity-io/falcon-docs/pull/68/commits/5dab62548573cb8472e134f42085146b38c5d85b#diff-d77539390862c1cf3fb230df4a7d0e3cR5

**Pull request merged**